drivers/windows/labels.go

```diff
@@ -30,4 +30,10 @@ const (
 
 	// SourceMac of the network
 	SourceMac = "com.docker.network.windowsshim.sourcemac"
+
+	// EnableICC label
+	EnableICC = "com.docker.network.bridge.enable_icc"
+
+	// DisableDNS label
+	DisableDNS = "com.docker.network.windowsshim.disable_dns"
```

Comment:
Embedded DNS is a libnetwork Core's property and ideally the driver should not worry about it. Since this flag is not used in this PR, it is hard to understand if it will be used from docker directly.